tests/testdata/programs/ts/rdsmod/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as rds from "@pulumi/rds";
+
+
+const testrds = new rds.Module("test-rds", {
```

Comment:
I think this would be a good opportunity to add a more complete test. For example we could base it off of [this complete mysql example](https://github.com/terraform-aws-modules/terraform-aws-rds/blob/master/examples/complete-mssql/main.tf), but modify it so that the resources and data resources are using pulumi resources. That would allow us to test a more real world scenario and would include

- Passing output values from pulumi resources into modules
- Passing outputs from one module to another
- Passing module outputs to Pulumi resources as inputs